### ISSUE TITLE
The application loses settings in the portable mode

### ISSUE THREAD

**Issue by leonid1812** [crow-translate/crow-translate#550]
I would like to bring to your attention the following issue regarding the persistence of application settings and the "Run in Portable Mode" option:

1. I updated the application settings.
2. I specifically selected the option to run the application in portable mode.
3. However, after rebooting my computer, the application lost the "Run in Portable Mode" option and automatically switched back to normal mode.
4. As a result, all of my settings were dropped and reset to their default values.

This issue has caused significant inconvenience, as I rely on the portable mode for seamless usage across different devices without the need for reconfiguration.

**Comment by Shatur** [comment]
Interesting bug. After selecting portable mode a settings file should appear in the Crow directory. Could you check if its there?

**Comment by leonid1812** [comment]
There are some screenst. (Windows 11 Pro, 64-bit)
1. Set the aaplication portable mode
![Settings01](https://github.com/crow-translate/crow-translate/assets/17238147/f2df0185-350d-4f85-875c-34c895a1be4f)
2. Ini-file created in the application home folder
![config_file](https://github.com/crow-translate/crow-translate/assets/17238147/47343a8d-5d44-4fed-9687-16a62bb04a1e)
3. Reboot PC
4. settings.ini is still exists in application home folder, but application ignored this file and started in not-portables mode (keep confins in C:\ driver)

P.S. Settings after reboot
![Settings_after reboot](https://github.com/crow-translate/crow-translate/assets/17238147/9fc10b96-cb09-4e1c-a332-43accb906dc6)

**Comment by Shatur** [comment]
Could you try to enable portable mode and exit the application normally?

**Comment by dmocha** [comment]
Updating to the latest portable version 2.10.6 loses settings. Selecting the "portable mode" field creates a settings.ini file, but not in the program directory, but on the main drive. In my case, the program is located in d:\tools\crow-translate\ and the settings file is created in d:\settings.ini
Disabling and enabling the portable version does not help.
Moving the settings.ini file doesn't help either.

**Comment by Shatur** [comment]
That's odd. @VolkMilit tried to fix this in #560. Looks like `QCoreApplication::applicationDirPath()` returns `D:\`...
I think it's related to static initialization We probably should defer it.

**Comment by VolkMilit** [comment]
That's really weired. [Just check the docs](https://doc.qt.io/qt-5/qcoreapplication.html#applicationDirPath), everything should be Ok. Looks like Qt bug to me.

```
Returns the directory that contains the application executable.

For example, if you have installed Qt in the C:\Qt directory, and you run the regexp example, this function will return "C:/Qt/examples/tools/regexp".
```
